### DISCUSSION TITLE
getting this error when trying to connect with rainbowkit with chain zkSync and argent, metamask works fine

### DISCUSSION THREAD

**Discussion by Morgosh** [rainbow-me/rainbowkit#1357] [Feedback]
<img width="232" alt="image" src="https://github.com/rainbow-me/rainbowkit/assets/35894423/4f52b5a1-505f-4024-abd3-ad40626a69e5">

**Comment by DanielSinclair** [comment]
This looks like a problem with the wallet specifically. WalletConnect will throw errors received from the wallet if i.e. chains are unsupported. @Morgosh which wallet were you using during this test?

**Reply by Morgosh** [reply]
Argent

**Reply by Morgosh** [reply]
Metamask worked, tho will need to upgrade the rainboekit now at retest it

**Reply by DanielSinclair** [reply]
This is likely an issue with Argent's implementation of WalletConnect v2. These types of issues should hopefully smooth out in the coming weeks. You'll want to ping their team if you're still experiencing the issue.
